lib/system/api/network.toit

```diff
@@ -159,249 +154,3 @@ class NetworkServiceClient extends ServiceClient implements NetworkService:
 
   socket_mtu handle/int -> int:
     return invoke_ NetworkService.SOCKET_MTU_INDEX handle
-
-class NetworkResource extends ServiceResourceProxy:
```

Comment:
The code in this file was moved without changes to lib/system/base/network.toit.